app/models/job.rb

```diff
@@ -0,0 +1,9 @@
+class Job < ApplicationRecord
+  belongs_to :company
+
+  has_rich_text :description
+
+  enum job_type: { 'Full-time': 0, 'Part-time': 1 }
```

Comment:
We won't be using strings for the keys of enums. 

```suggestion
  enum job_type: { full_time: 0, part_time: 1 }
```